src/Lykke.Service.HFT/Controllers/OrdersController.cs

```diff
@@ -206,47 +228,20 @@ public class OrdersController : Controller
             {
                 return Ok();
             }
-            // todo: produce valid http status code if status is 'Matched' or 'Pending'
 
             var response = await _matchingEngineAdapter.CancelLimitOrderAsync(id);
             if (response.Error != null)
-            {
-                // todo: produce valid http status codes based on ME response 
+            { 
                 return BadRequest();
```

Comment:
Wouldn't it be better to return some details like you are doing in other methods when returning BadRequest?

Reply:
Sure; fixed.